bin/codecov.sh

```diff
@@ -19,11 +19,11 @@ for d in $(go list ./... | grep -v vendor); do
       continue
     fi
 
-    go test -coverprofile=${PROF} $d
+    # do not stop even when a test fails
```

Comment:
It would be nice to see this as a separate PR.  I find the fact that code coverage bombs out on one failure to be a little annoying :)

Reply:
`parallel-codecov.sh` seems to be used primarily although `codecov.sh` still exists, and the parallel one does not suffer from that issue.